core/vm/eips.go

```diff
@@ -320,3 +321,26 @@ func enable7516(jt *JumpTable) {
 		numPush:     1,
 	}
 }
+
+func enable7702(jt *JumpTable) {
+	jt[EXTCODECOPY].constantGas = params.WarmStorageReadCostEIP2929
```

Comment:
Any reason to also update this constant gas which was also set in `enable2929`?

Reply:
just making it explicit. No other reason.

Reply:
Better to just put down the changes only